docs/standard/architecture/architecting-container-and-microservice-based-applications/container-design-principles.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Container design principles | Microsoft Docs 
+description: .NET Microservices Architecture for Containerized .NET Applications | Container design principles
+keywords: Docker, Microservices, ASP.NET, Container
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 05/19/2017
+---
```

Comment:
Let us know when this is ready to review. One thing I've noticed is that it's missing the required ms.prod value and the optional ms.technology. 
I could ask Katya to add a value of dotnet-docker for ms.technology to the .net ms.prod product family (we have for .net-core and .net-framework already).